Library/Formula/dovecot.rb

```diff
@@ -32,36 +32,38 @@ def install
     system "make", "install"
   end
 
-  def caveats; <<-EOS.undent
-      For Dovecot to work, you will need to do the following:
-
-      1) Create configuration in #{etc}
-
-      2) If required by the configuration above, create a dovecot user and group
+  plist_options :startup => true
 
-      3) possibly create a launchd item in /Library/LaunchDaemons/#{plist_path.basename}, like so:
-      <?xml version="1.0" encoding="UTF-8"?>
-      <!DOCTYPE plist PUBLIC "-//Apple Computer//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
-      <plist version="1.0">
+  def plist; <<-EOS.undent
+    <?xml version="1.0" encoding="UTF-8"?>
+    <!DOCTYPE plist PUBLIC "-//Apple Computer//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+    <plist version="1.0">
       <dict>
-              <key>Label</key>
-              <string>#{plist_name}</string>
-              <key>OnDemand</key>
-              <false/>
-              <key>ProgramArguments</key>
-              <array>
-                      <string>#{HOMEBREW_PREFIX}/sbin/dovecot</string>
-                      <string>-F</string>
-              </array>
-              <key>RunAtLoad</key>
-              <true/>
-              <key>ServiceDescription</key>
-              <string>Dovecot mail server</string>
+        <key>Label</key>
+        <string>#{plist_name}</string>
+        <key>OnDemand</key>
+        <false/>
+        <key>RunAtLoad</key>
+        <true/>
+        <key>ProgramArguments</key>
+        <array>
+          <string>#{opt_sbin}/dovecot</string>
+          <string>-F</string>
+        </array>
+        <key>ServiceDescription</key>
+        <string>Dovecot mail server</string>
       </dict>
-      </plist>
+    </plist>
+    EOS
+  end
 
-      Source: http://wiki.dovecot.org/LaunchdInstall
-      4) start the server using: sudo launchctl load /Library/LaunchDaemons/#{plist_path.basename}
+  def caveats; <<-EOS.undent
+    For Dovecot to work, you may need to create a dovecot user
+    and group depending on your configuration file options.
     EOS
   end
+
+  test do
```

Comment:
It's a profoundly bad test, but servers are a complete nightmare to test :smile_cat: